src/externalized/DefaultContentManager_unittests.cc

```diff
@@ -127,4 +137,37 @@ TEST(TempFiles, deleteFile)
 	delete cm;
 }
 
+TEST(ExternalizedData, readAllData)
+{
+	DefaultContentManager* cm = DefaultContentManagerUT::createDefaultCMForTesting();
+	ASSERT_TRUE(cm->loadGameData());
+}
+
+std::vector<ST::string> recursivelyListAllFiles(ST::string dir, std::string suffix)
+{
+	std::vector<ST::string> files;
+	for (auto& p : fs::recursive_directory_iterator(dir.to_std_string()))
```

Comment:
We have nothing in FileMan or our rust fs layer to do this? The whole std::filesystem include block looks fragile. CC @flaviojs

Reply:
How about now? Added `ls -R` to FileMan

Reply:
Looks better to me, thanks. If this is now done and Flávio doesn't comment in a while, go ahead and merge it.